### ISSUE TITLE
UserWarning: None of the inputs have requires_grad=True. Gradients will be None

### ISSUE THREAD

**Issue by Chsengni** [We-IOT/chatglm3_6b_finetune#20]
![image](https://github.com/We-IOT/chatglm3_6b_finetune/assets/31051072/97d7e454-5ece-4807-b436-d4f057099543)

**Comment by Chsengni** [comment]
![image](https://github.com/We-IOT/chatglm3_6b_finetune/assets/31051072/98782cf2-f296-4631-bbb3-0e6cb2a661a4)
这个什么情况啊

**Comment by Chsengni** [comment]
python 3.10.10

**Comment by Chsengni** [comment]
明白了，json文件格式整错了

**Comment by Chsengni** [comment]
a

**Comment by Linuxstyle** [comment]
我也遇到相同的 问题了，请问你这个是怎么解决的呢？

**Comment by huang199408** [comment]
请问一下是怎么解决的，我的Python是3.10.16，不知道是不是这个问题
